app/queries/find_ids_with_property_set.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class FindIdsWithPropertySet
```

Comment:
`Property Set` is a little confusing because it could mean a `Set` like the noun. I know it's longer but maybe we could call it `FindIdsWithPropertyNotNil`? Or something else that's more specific? I think `FindIdsWithProperty` might even work.